man/varlinkctl.xml

```diff
@@ -61,6 +61,17 @@
       <arg choice="opt"><replaceable>ARGUMENTS</replaceable></arg>
     </cmdsynopsis>
 
+    <cmdsynopsis>
+      <command>varlinkctl</command>
+      <arg choice="opt" rep="repeat">OPTIONS</arg>
+      <arg choice="plain">--exec call</arg>
+      <arg choice="plain">call</arg>
+      <arg choice="plain"><replaceable>ADDRESS</replaceable></arg>
+      <arg choice="plain"><replaceable>METHOD</replaceable></arg>
+      <arg choice="plain"><replaceable>ARGUMENTS</replaceable></arg>
+      <arg choice="plain">--</arg>
```

Comment:
Maybe better to append 
`<arg choice="plain"><replaceable>COMMAND</replaceable></arg>`
at the end ?